syntax = "proto3";

option csharp_namespace = "Unity.MLAgents.CommunicatorObjects";
package communicator_objects;

message AgentActionProto {
    repeated float vector_actions = 1;
    reserved 2; // deprecated string text_actions = 2;
    reserved 3; //deprecated repeated float memories = 3;
    float value = 4;
    reserved 5; // deprecated CustomActionProto custom_action = 5;
}
